backend/settings.py

```diff
@@ -218,6 +222,11 @@
         "default_link_target": "_blank",
     }
 
+# Watchman settings (https://github.com/mwarkentin/django-watchman)
+WATCHMAN_TOKENS = os.getenv('DJANGO_WATCHMAN_TOKENS', None)
+WATCHMAN_TOKEN_NAME = os.getenv('DJANGO_WATCHMAN_TOKEN_NAME', 'token')
```

Comment:
Optional: I feel the `token` name is too generic. I would make it `cae_wm_token` or `wm_token`

Reply:
Okay, I'll change it.